source/main/physics/BeamForcesEuler.cpp

```diff
@@ -472,6 +472,7 @@ void Actor::CalcWheels(bool doUpdate, int num_steps)
         ar_engine->SetWheelSpin(ar_wheel_spin * RAD_PER_SEC_TO_RPM); // Update the driveshaft speed
     }
 
+#if 0

```

Comment:
NEEDINFO - what is the issue here?

Reply:
It segfaults in some cases. I guess I should have taken the time to move it into the GfxActor?

Reply:
> in some cases.

On shutdown or exiting to main menu I bet. Otherwise it wouldn't make sense. OpenAL by principle runs it's own thread, thus it's sound-update functions are lightweight and can be called anywhere anytime. It doesn't belong to GfxActor. If it segfaults, it's our bug.

Reply:
No, on truck activation.

Reply:
> can be called anywhere anytime

But we shouldn't do any sound updates from inside the physics loop in my opinion.

Reply:
-> https://github.com/RigsOfRods/rigs-of-rods/commit/8e0c8ac3c487ec31d8d05afd4856aeaab7110f1b